go/tests/prog/range_prog/main.go

```diff
@@ -22,8 +20,14 @@ func main() {
 		fmt.Println(err)
 		return
 	}
-	for _, i := range lib.IntRange(a, b) {
-		fmt.Print(i)
+	fmt.Print(a)
+	for b != a {
+		if a < b {
+			a++
+		} else {
+			a--
+		}
+		fmt.Print(" ", a)
```

Comment:
If you tested it and are 100%-ish sure that it works, you can push/merge master directly

Reply:
ok i will merge it then, thank you